softwarereview_policies.Rmd

```diff
@@ -97,12 +95,6 @@ Packages should be *general* in the sense that they should solve a problem as br
 
 Packages that include interactive tools to facilitate researcher workflows (e.g., shiny apps) must have a mechanism to make the interactive workflow reproducible, such as code generation or a scriptable API.
 
-Here are some types of packages we are unlikely to accept:
-
--  Packages that wrap or implement statistical or machine learning methods.  We are not organized so as to review the correctness of these methods (But see "text analysis", above).
--  Exploratory data analysis packages that visualize or summarize data.
```

Comment:
I'd still keep the two last examples?

Reply:
But they're both stats packages and so can be accepted?

Reply:
ooooh